lib/games/games_module.rb

```diff
@@ -17,19 +16,44 @@ def add_game
     print 'Please state how many years ago it was published: '
     published_date = gets.chomp
     new_game = Game.new(title, multiplayer, last_played_at, published_date)
-    @game_list.push(new_game)
+    @game_list.push(title: new_game.title, multiplayer: new_game.multiplayer, last_played_at: new_game.last_played_at, published_date: new_game.published_date)
     puts 'Game created succesfully'
     puts ''
   end
 
   def list_all_games
-    @game_list.each do |game|
-      puts "Title: \"#{game.title}\", Multiplayer: #{game.multiplayer}, Last Time Played:#{game.last_played_at} years ago, Published #{game.published_date} years ago"
+    @game_list = JSON.parse(File.read('./lib/games/json_data/games.json'))
```

Comment:
I suggest adding a validation or wrapping this in a begin/rescue/else block. When the File doesn't exist it throws an Error and the Application Stops. Another suggestion is making a new method to load/read a state from a file this method should list_all_games so a person that doesn't know your code would expect that this only list some items from an array/hash without modifying the original data structure.

A simple way to do it without a Begin/Rescue Block is something like this:

```
  if File.exist?('./lib/games/json_data/games.json')
        return JSON.parse(File.read('./lib/games/json_data/games.json'))
  end
```

Error: When you add a Game and try to list_all_games you receive an error if that file is not present in the project folder.

 
![image](https://user-images.githubusercontent.com/85680436/144166408-1de888a0-f9b2-410e-857f-68cfc83886de.png)